Bottom: 4d95d77cb4a52738759b19f6e33a635077ec3262
Top:    2b6b42daa54b3fc2b6a71a00e723eea6271da71e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-06 12:19:54 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
